smarts/core/remote_agent_buffer.py

```diff
@@ -196,7 +196,9 @@ def get_manager_channel_stub(addr):
     try:
         # Wait until the grpc server is ready or timeout after 30 seconds
         grpc.channel_ready_future(channel).result(timeout=30)
-    except grpc.FutureTimeoutError:
-        raise RemoteAgentException("Timeout in connecting to remote zoo manager.")
+    except grpc.FutureTimeoutError as e:
+        raise RemoteAgentException(
+            "Timeout in connecting to remote zoo manager."
+        ) from e
```

Comment:
@Adaickalavan already fixed this.

Reply:
Here, the `grpc.FutureTimeoutError` may only mean one thing, which is that we are unable to connect to the server due to a timeout. 

Since there is no other information to be gained from this error, and to keep the error message concise and helpful to the user, I believe there is no need to print out the actual timeout error using `from e` which does not look very beautiful.

The originally fixed code is fine:
```
except grpc.FutureTimeoutError:
    raise RemoteAgentException("Timeout in connecting to remote zoo manager.")
```